flygym/examples/hybrid_controller.py

```diff
@@ -179,41 +208,40 @@ def run_hybrid_simulation(sim, cpg_network, preprogrammed_steps, run_time):
         for segment in ["Tibia", "Tarsus1", "Tarsus2", "Tarsus3", "Tarsus4", "Tarsus5"]
     ]
 
-    np.random.seed(0)
-
+    # Initialize the simulation
     fly = Fly(
         enable_adhesion=True,
         draw_adhesion=True,
+        init_pose="stretch",
+        control="position",
         contact_sensor_placements=contact_sensor_placements,
     )
+    cam = Camera(fly=fly, play_speed=0.1, camera_id=f"{fly.name}/camera_right")
 
-    cam = Camera(fly=fly, play_speed=0.1, camera_id="Animat/camera_right")
     sim = SingleFlySimulation(
         fly=fly,
         cameras=[cam],
-        timestep=1e-4,
-        arena=GappedTerrain(),
+        timestep=timestep,
+        arena=MixedTerrain(),
     )
 
+    # run cpg simulation
+    obs, inf = sim.reset()
     cpg_network = CPGNetwork(
-        timestep=1e-4,
+        timestep=timestep,
         intrinsic_freqs=intrinsic_freqs,
         intrinsic_amps=intrinsic_amps,
         coupling_weights=coupling_weights,
         phase_biases=phase_biases,
         convergence_coefs=convergence_coefs,
-        seed=0,
     )
 
-    obs, info = sim.reset()
     print(f"Spawning fly at {obs['fly'][0]} mm")
 
-    obs_list, had_physics_error = run_hybrid_simulation(
+    obs_list, inf_list, had_physics_error = run_hybrid_simulation(
```

Comment:
Let's change `inf_list` to `info_hist` since (1) `inf` means infinity in numpy and (2) I used `info_hist` elsewhere.

Similarly, `obs_list` -> `obs_hist` just for consistency.